apps/docs-snippets2/src/types/address/address-5.ts

```diff
@@ -0,0 +1,10 @@
+// #region full
+import { Address } from 'fuels';
+
+const dataFromInput: string =
```

Comment:
In this case of breaking snippets apart, should we use more _descriptive_ names?

I suspect numerical indexes will not be very straightforward.

Descriptive names can make it easier to locate snippets.

Something like:
 - `fromDynamicInput.ts`
 - `fromString.ts`

Or:
 - `from-dynamic-input.ts`
 - `from-string.ts`

Instead of:
 - `address-1.ts`
 - `address-2.ts`